### DISCUSSION TITLE
unsupported tarball input attribute 'lastModified'

### DISCUSSION THREAD

**Discussion by behaghel** [juspay/nix-dev-home#108] [Q&A] [answered]
I get this error below after using the template verbatim with `nix run`. I was initially migrating manually my config to nixos-unified  and was stuck on the same error when I decided to stand on solid ground with the template. 

```
❯ nix run
warning: unknown experimental feature 'configurable-impure-env'
2024-11-02T23:59:29.326|INF|currentSystem=aarch64-darwin currentHost=tfmbp targetHost=tfmbp hostData={outputs: {nixArgs: [], overrideInputs: {}, system: aarch64-darwin}, overrideInputs: [], sshTarget: , host: tfmbp, flake: /nix/store/as724w61zqb09vjqma6zalxcizxazzwv-nixos-unified-activate-flake#tfmbp}
2024-11-02T23:59:29.330|INF|Activating locally
2024-11-02T23:59:29.330|INF|>>> darwin-rebuild switch --flake /nix/store/as724w61zqb09vjqma6zalxcizxazzwv-nixos-unified-activate-flake#tfmbp
warning: unknown experimental feature 'configurable-impure-env'
error:
       … while updating the lock file of flake 'path:/nix/store/as724w61zqb09vjqma6zalxcizxazzwv-nixos-unified-activate-flake?lastModified=1&narHash=sha256-Svd7CjSPyI7dBerMp73ujU6SykpxRJZCrjx4Q%2BkL4pU%3D'

       error: unsupported tarball input attribute 'lastModified'
```

**Comment by srid** [comment]
@behaghel What is your Nix version?

**Reply by behaghel** [reply]
Hi @srid, initially with `2.18.1`, I then suspected something and tried with version `2.24.10`. Both display identical behaviour as reported above.

**Reply by behaghel** [reply, marked as answer]
I have uninstalled and reinstalled nix entirely on this MBP and it seems to have helped. Will report in a few.

**Reply by behaghel** [reply]
Indeed. That reinstall made the problem disappear. Thanks for your support!

**Reply by srid** [reply]
Perhaps you had two different versions of Nix installed? I recall this issue appearing with older Nix.
